### PR TITLE
Fix for compare sequence numbers and online help

### PR DESCRIPTION
**Pull request by KevinLoesch1**
**What It Does**
This pull request corrects two bugs. The first one is to correct the output and compare when **zowe files compare** is issued on Windows and the **--noseq false** option is specified. The second fix is for the **zowe files compare** online help. 

https://github.com/zowe/zowe-cli/issues/1754
https://github.com/zowe/zowe-cli/issues/1992

**How to Test**
Run **zowe files compare** with the **--noseq false** on Windows against files that have sequence numbers. The first sequence number should not be the last character of the line. 

**Review Checklist**
I certify that I have:
- [X] tested my changes
- [X] added/updated automated tests
- [X] updated the changelog
- [X] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by gejohnston — APPROVED** [review]

File: packages/cli/__tests__/zosfiles/__unit__/compare/ds/Dataset.handler.unit.test.ts [hunk 1]
```diff
@@ -178,6 +178,41 @@ describe("Compare data set handler", () => {
             expect(getDiffStringSpy).toHaveBeenCalledWith("compared", "compared", options);
         });
 
+        it("should compare two data sets containing carriage returns in terminal with --seqnum specified(Windows Specific)", async () => {
+            if(process.platform === "win32")
```
Comment (zFernand0) [inline packages/cli/__tests__/zosfiles/__unit__/compare/ds/Dataset.handler.unit.test.ts:182]: should we do `(process.platform === "win32" ? it : it.skip)("should compare ...` ?
That way we don't have an empty test on other operating systems?

I don't recall if this is the exact syntax though.

**Review by zFernand0 — APPROVED** [review]
Left a small comment, but changes LGTM! 😋

File: packages/cli/src/zosfiles/compare/CompareBaseHelper.ts [hunk 2]
```diff
@@ -164,7 +164,11 @@ export class CompareBaseHelper {
     public prepareContent(content: string | Buffer): string {
         let contentString = content.toString();
         if(this.seqnum === false) {
-            const seqnumlen = 8;
+            let seqnumlen = 8;
+            /* If Windows, account for the carriage return */
+            if(process.platform === "win32" && content.toString().endsWith("\r\n")){
```
Comment (t1m0thyj) [inline packages/cli/src/zosfiles/compare/CompareBaseHelper.ts:169]: Is checking for Windows necessary? It's possible to have a local file with Windows line endings (CRLF) on any OS.

Comment (t1m0thyj) [reply packages/cli/src/zosfiles/compare/CompareBaseHelper.ts:169]: I tried commenting out the process check and testing on my Mac with a local file that has CRLF. It seems like the first sequence number still shows up:
```
zowe files compare lf-uss temp.jcl "/tmp/XXX/iefbr14.jcl" --no-seqnum
- Removed
+ Added

- //IEFBR14J JOB  (IZUACCT),'TIMOTHY',CLASS=A                             0
- //STEP1    EXEC PGM=IEFBR14                                             0
+ //IEFBR14J JOB  (IZUACCT),'TIMOTHY',CLASS=A                             
+ //STEP1    EXEC PGM=IEFBR14                                             
```

Comment (KevinLoesch1) [reply packages/cli/src/zosfiles/compare/CompareBaseHelper.ts:169]: I was thinking about that possibility. I will make the change.

**Pull request merged**